src/store/juju/slice.ts

```diff
@@ -139,6 +142,9 @@ const slice = createSlice({
       state.auditEvents.loaded = false;
       state.auditEvents.loading = false;
     },
+    updateAuditEventsLimit: (state, { payload }: PayloadAction<number>) => {
```

Comment:
This will need a test in https://github.com/canonical/juju-dashboard/blob/audit-logs/src/store/juju/actions.test.ts